### DISCUSSION TITLE
Is there a permalink for the Winget dependencies zip file?

### DISCUSSION THREAD

**Discussion by HotCakeX** [microsoft/winget-cli#4937] [Q&A] [answered]
I use Winget on GitHub runners which are Windows Servers. Since it's not preinstalled by default (!), i have [a setup script](https://github.com/HotCakeX/Harden-Windows-Security/blob/main/.github/workflows/Build%20AppControl%20Manager%20MSIX%20Package.yml). I use it to download Winget and its dependencies, one of the dependencies is this:
```
https://aka.ms/Microsoft.VCLibs.x64.14.00.Desktop.appx
```

Since the last Winget version, i get error when trying to install it, saying i need VCLibs version `14.0.33728.0`, but that aka.ms link only hosts version `14.0.33321.0`.

<br>

Alternatively, please consider updating the dependency link in aka.ms to the latest version whenever a new Winget version is released or dependencies change. Thanks

https://learn.microsoft.com/en-us/troubleshoot/developer/visualstudio/cpp/libraries/c-runtime-packages-desktop-bridge

<br>

**Comment by SpecterShell** [comment]
The dependencies have been published along with the latest version. You can find them in the file "DesktopAppInstaller_Dependencies.zip".

Also see: https://github.com/microsoft/winget-cli/pull/4923

**Comment by vedantmgoyal9** [comment, marked as answer]
https://github.com/microsoft/winget-cli/releases/latest/download/DesktopAppInstaller_Dependencies.zip
